web/src/routes/Alerts.tsx

```diff
@@ -0,0 +1,22 @@
+import { Alert, AlertActionCloseButton, AlertGroup } from "@patternfly/react-core";
+import { useRecoilState } from "recoil";
+import { alertState } from "src/atoms/AlertState";
+
+
+export default function Alerts(){
+    const [alerts, setAlerts] = useRecoilState(alertState);
+    return (
+    <AlertGroup isToast isLiveRegion>
+        {alerts.map(alert=><Alert
```

Comment:
what do you think about adding a [timeout](https://www.patternfly.org/v4/components/alert/#alert-timeout) for alerts?

Reply:
I was thinking we would want to keep an error alert so the user has a chance to debug. It makes sense for successful messages though.